### PR TITLE
Adds React scaffold

### LINKED ISSUE #3: User Story 1 - Landing Page
As a user, When the page loads a landing page asking if I want to play

### PR DESCRIPTION
**Pull request by ajtran303**
Screen Shot 2020-11-14 at 12 27 05 PM

This PR adds the scaffold generated by `create-react-app`, and some nominal changes to make it look like we aren't using the default template.

I've also updated the Readme.

This will close #3 because technically this is the landing page, there is a button to play, and there is a test for it.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/quizzery-icon.png" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Quizzery is a fast-paced game full of trivia challenges and questions. Quizzery will test your knowledge of world history, popular culture, sports, TV shows, movies, geography, music and more!"
+    />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
```
Comment (ajtran303) [inline public/index.html:12]: This icon is still the React icon, I am not sure if it should be changed -- what is `apple-touch-icon` ??
Can this line just be safely deleted?

File: public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "Quizzery",
+  "name": "Quizzery: A Jeopardy-style quiz platform",
+  "icons": [
```
Comment (ajtran303) [inline public/manifest.json:4]: Not a high priority to investigate but we should learn about this section of the manifest

File: src/App.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="Quizzery banner logo" />
+        <a
+          className="App-link"
+          href="https://reactjs.org"
```
Comment (ajtran303) [inline src/App.js:11]: This will have to change

File: src/App.test.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+// 
+// test('renders learn react link', () => {
+//   render(<App />);
+//   const linkElement = screen.getByText(/learn react/i);
+//   expect(linkElement).toBeInTheDocument();
+// });
+
+test('renders start button', () => {
```
Comment (ajtran303) [inline src/App.test.js:10]: This technically completes the user story #3

File: src/index.css [hunk 5]
```diff
@@ -0,0 +1,13 @@
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
```
Comment (ajtran303) [inline src/index.css:3]: One of these could be cool to use in our UI!

File: src/index.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```
Comment (ajtran303) [inline src/index.js:1]: It seems like we're just going to leave this file alone.

**Review by ajtran303 — COMMENTED** [review]
Regarding styling: maybe on a new branch we can remove all the current styling to make way for the Bootstrap components we would like to use.

File: src/index.js (hunk 6 above)
Comment (jKwibe) [reply src/index.js:1]: Yap! I bet we will have to import react in all components too.

File: src/App.js (hunk 3 above)
Comment (jKwibe) [reply src/App.js:11]: For sure! We have to get rid of all react boilerplate code.

**Pull request merged**